packages/pixeloven-tasks/src/middleware/errorHandler.ts

```diff
@@ -67,9 +67,9 @@ function renderProblems(type: string, lines: string[]) {
     lines.forEach(msg => problems.push(renderProblem(type, msg)));
     return (
         "<html>" +
-        "<body style=\"" +
+        '<body style="' +
```

Comment:
What's with the mix of quote style here?

Reply:
Ugh, prettier or something keeps changing it back. I have a ticket to look into that stuff